tests/letstest/scripts/test_leauto_upgrades.sh

```diff
@@ -64,41 +64,7 @@ iQIDAQAB
 -----END PUBLIC KEY-----
 "
 
-if [ $(python -V 2>&1 | cut -d" " -f 2 | cut -d. -f1,2 | sed 's/\.//') -eq 26 ]; then
```

Comment:
We should keep these lines as they are testing our behavior of installing Python 3 in certbot-auto on systems that still use Python 2.6.